source/modules/MEI.stringtab.xml

```diff
@@ -77,7 +77,13 @@
           <rng:ref name="data.VERTICALALIGNMENT"/>
         </datatype>
       </attDef>
-      <!-- Below here is unchecked / legacy -->
+      <attDef ident="tab.anchorline" usage="opt">
+        <desc xml:lang="en">Used in German lute tablature where the vertical alignment of notes is consistent but cannot be identified using a typical value of <att>tab.align</att> (<abbr>i.e.</abbr>, <val>top</val> or <val>bottom</val>). Specifies the horizontal strand corresponding to the <att>lines</att> attribute on <gi scheme="MEI">staffDef</gi> that anchors the vertical position of notes. This anchorline is used as the vertical starting position when stacking notes into chords. Single notes simply occupy this position. Chords 'grow upwards' from this position. If the chord extends further than the number of available horizontal strands ('lines') above the anchorline, the entire chord is shifted downward until its top note is positioned on the top-most line. (Note that in German lute tablature, the lines are conceptual rather than visible).
```

Comment:
```suggestion
        <desc xml:lang="en">Used in German lute tablature where the vertical alignment of notes is consistent but cannot be identified using a typical value of <att>tab.align</att> (<abbr>i.e.</abbr>, <val>top</val> or <val>bottom</val>). Specifies the horizontal strand corresponding to the <att>lines</att> attribute on <gi scheme="MEI">staffDef</gi> that anchors the vertical position of notes. This anchorline is used as the vertical starting position when stacking notes into chords. Single notes simply occupy this position. Chords <q>grow upwards</q> from this position. If the chord extends further than the number of available horizontal strands (<q>lines</q>) above the anchorline, the entire chord is shifted downward until its top note is positioned on the top-most line. (Note that in German lute tablature, the lines are conceptual rather than visible).
```

Reply:
Sorry, one more comment, cf. #1498